Sources/firebase/include/FirebaseCore.hh

```diff
@@ -3,7 +3,7 @@
 
 #include <firebase/util.h>
 
-#include <stdio.h>
```

Comment:
Yeah, that was leftover from some debugging I was doing. Oops!

Reply:
Well, we also didn't catch it during review, so double oops! 😂